.github/workflows/build_apk.yml

```diff
@@ -0,0 +1,44 @@
+name: Sample GitHub Actions Workflow
+on:
+  push:
+    branches:
+      - master
+
+  workflow_dispatch:
+
+jobs:
+  avni-apk:
```

Comment:
You can also better name the job. `build-apk` something like that

Reply:
yes modified